src/components/Components/LinkedSecretsListView/LinkedSecretsListRow.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, Flex, Label, Text, TextVariants } from '@patternfly/react-core';
+import { RowFunctionArgs, TableData } from '../../../shared';
+import { SecretKind } from '../../../types';
+import { linkedSecretsTableColumnClasses } from './LinkedSecretsListHeader';
+
+import './LinkedSecretsListView.scss';
+
+export const LinkedSecretsListRow: React.FC<RowFunctionArgs<SecretKind>> = ({ obj: secret }) => {
+  return (
+    <>
+      <TableData className={linkedSecretsTableColumnClasses.secretName}>
+        <Text component={TextVariants.p}>{secret.metadata.name}</Text>
+      </TableData>
+
+      <TableData className={linkedSecretsTableColumnClasses.type}>
+        <Text component={TextVariants.p}>{secret.type}</Text>
+      </TableData>
+
+      <TableData className={linkedSecretsTableColumnClasses.labels}>
+        {secret.metadata.labels ? (
+          Object.keys(secret.metadata.labels ?? {}).map((key) => (
+            <Label
+              key={key}
+              className="linked-secrets-list-view__label"
+            >{`${key}=${secret.metadata.labels[key]}`}</Label>
+          ))
+        ) : (
+          <Text component={TextVariants.p}> - </Text>
+        )}
+      </TableData>
+
+      <TableData className={linkedSecretsTableColumnClasses.actions}>
+        {/* TODO: onClick logic will be implemented in another ticket*/}
```

Comment:
the "unlink" logic is going to be implemented on this ticket https://issues.redhat.com/browse/KFLUXUI-421 :)